client/daemon.go

```diff
@@ -0,0 +1,43 @@
+package main
+
+import (
+	"os"
+	"os/exec"
+	"syscall"
+)
+
+const daemonBinary = "dockerd"
+
+// DaemonProxy acts as a cli.Handler to proxy calls to the daemon binary
+type DaemonProxy struct{}
+
+// NewDaemonProxy returns a new handler
+func NewDaemonProxy() DaemonProxy {
+	return DaemonProxy{}
+}
+
+// CmdDaemon execs dockerd with the same flags
+// TODO: add a deprecation warning?
```

Comment:
I think we may have found a reasons to deprecated it.

Windows doesn't have `syscall.Exec` apparently, so `docker daemon` isn't going to work on Windows. I think we need this to report an error saying to call `dockerd` directly. We probably don't want to keep linux and windows engine different, so I think it would be good to log a warning here saying to use `dockerd` directly. That way they will converge eventually.